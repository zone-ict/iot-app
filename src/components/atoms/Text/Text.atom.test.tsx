import React from 'react';
import renderer from 'react-test-renderer';
import Text from './Text.atom';

describe('Text component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text>Text</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Text.HeadingOne component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text.HeadingOne>Text</Text.HeadingOne>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Text.HeadingTwo component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text.HeadingTwo>Text</Text.HeadingTwo>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Text.HeadingThree component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text.HeadingThree>Text</Text.HeadingThree>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Text.HeadingFour component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text.HeadingFour>Text</Text.HeadingFour>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Text.HeadingFive component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Text.HeadingFive>Text</Text.HeadingFive>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
