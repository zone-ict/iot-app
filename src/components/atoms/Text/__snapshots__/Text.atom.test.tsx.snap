// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component should render correctly 1`] = `
<div
  className="css-onny4r"
>
  Text
</div>
`;

exports[`Text.HeadingFive component should render correctly 1`] = `
<h5
  className="css-14a2izr"
>
  Text
</h5>
`;

exports[`Text.HeadingFour component should render correctly 1`] = `
<h4
  className="css-1h3xd9j"
>
  Text
</h4>
`;

exports[`Text.HeadingOne component should render correctly 1`] = `
<h1
  className="css-a15oo7"
>
  Text
</h1>
`;

exports[`Text.HeadingThree component should render correctly 1`] = `
<h3
  className="css-176q8bd"
>
  Text
</h3>
`;

exports[`Text.HeadingTwo component should render correctly 1`] = `
<h2
  className="css-jzboh9"
>
  Text
</h2>
`;
